setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = pytest-astropy
+url = https://astropy.org
+author = The Astropy Developers
+author_email = astropy.team@gmail.com
+classifiers =
+    Development Status :: 3 - Alpha
+    Framework :: Pytest
+    Framework :: Hypothesis
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development :: Testing
+    Topic :: Utilities
+license = BSD
+description = Meta-package containing dependencies for testing
+long_description = file: README.rst
+keywords = pytest, py.test, remotedata, openfiles, doctestplus
+
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+setup_requires = setuptools_scm
+install_requires =
+    pytest>=4.6
+    pytest-doctestplus>=0.2.0
+    pytest-remotedata>=0.3.1
+    pytest-openfiles>=0.3.1
+    pytest-astropy-header>=0.1.2
+    pytest-arraydiff>=0.1
+    pytest-filter-subpackage>=0.1
+    pytest-mpl>=0.11
+    pytest-cov
+    # Bump hypothesis to >=5.0 in early Jan 2020
+    hypothesis>=4.53
```

Comment:
I think there's a decent argument for `>=5.1` for [timezone support](https://hypothesis.readthedocs.io/en/latest/changes.html#v5-1-0) and generally avoiding possible deprecated behaviour :smile:

Reply:
Ok, I've bumped this - thanks!